File: 2414. Length of the Longest Alphabetical Continuous Substring.cpp
class Solution
{
public:
    int longestContinuousSubstring(string s)
    {
        int ans = 1, cnt = 1, n = s.size();
        for (int i = 1; i < n; i++)
        {
            if (s[i] == s[i - 1] + 1)
                cnt++;
            else
                cnt = 1;
            ans = max(ans, cnt);
        }
        return ans;
    }
};

File: 2414. Length of the Longest Alphabetical Continuous Substring.py
class Solution(object):
    def longestContinuousSubstring(self, s):
        n = len(s)
        ans, cnt = 1, 1
        for i in range(1, n):
            if ord(s[i]) == ord(s[i - 1]) + 1: 
                cnt += 1
            else:
                cnt = 1
            ans = max(ans, cnt)
        return ans